pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -2593,14 +2589,15 @@ FROM
 WHERE registered_entry_id IN (SELECT id FROM listing)
 
 ORDER BY selector_id, dns_name_id
-;`
-	return query, []any{entryID}, nil
+;`, buildQuestions(entryIDs))
+
+	return query, buildArgs(entryIDs), nil
 }
 
-func buildFetchRegistrationEntryQueryPostgreSQL(entryID string) (string, []any, error) {
-	const query = `
+func buildFetchRegistrationEntryQueryPostgreSQL(entryIDs []string) (string, []any, error) {
+	query := fmt.Sprintf(`
 WITH listing AS (
-	SELECT id FROM registered_entries WHERE entry_id = $1
+	SELECT id FROM registered_entries WHERE entry_id IN (%s)
```

Comment:
Do you know if this is likely to affect the performance of fetching a single way in any meaningful way? I imagine it shouldn't, they're equivalent.

Reply:
The AI chat bot says there is no significatnt performance difference: 
<img width="671" alt="Screenshot 2025-03-14 at 11 38 34 AM" src="https://github.com/user-attachments/assets/91856f7d-6fd2-4672-bd32-933732a4726d" />